Tests/AsyncAwait/ChannelAsyncIntegrationTests.swift

```diff
@@ -46,13 +46,13 @@ class ChannelIntegrationTests: BaseAsyncIntegrationTestCase {
   }
 
   func testChannelAsync_Delete() async throws {
-    try await channel.delete(soft: false)
+    _ = try await channel.delete(soft: false)
```

Comment:
Why @discardableResult was removed for delete method?

Reply:
What `@discardableResult` does is ignoring the return value from a method. Thus, you won't get the warning in the call below that the returned value isn't used:

```
try await channel.delete()
```

I realized this might be a trap for a caller, what you should do is to replace the existing reference with the updated value returned from the server. Keeping a `@discardableResult` silently ignores the fact that there's a return value, which might contain important information you should rely on